crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -407,31 +409,135 @@ fn entity_props_ui(
+    let Some(pinhole_ent_path) = pinhole_ent_path else { return; };
+
+    entity_props.backproject_pinhole_ent_path = Some(pinhole_ent_path.clone());
+
+    let tensor = query_latest_single::<Tensor>(&ctx.log_db.entity_db, entity_path, &query);
+    if tensor.as_ref().map(|t| t.meaning) == Some(TensorDataMeaning::Depth) {
+        let tensor = tensor.as_ref().unwrap();
+
+        ui.checkbox(&mut entity_props.backproject_depth, "Backproject Depth")
+            .on_hover_text(
+                "If enabled, the depth texture will be backprojected into a point cloud rather \
+                than simply displayed as an image.",
+            );
+        ui.end_row();
+
+        if entity_props.backproject_depth {
+            ui.label("Pinhole");
+            ctx.entity_path_button(ui, None, &pinhole_ent_path)
+                .on_hover_text(
+                    "The entity path of the pinhole transform being used to do the backprojection.",
+                );
+            ui.end_row();
+
+            // Compute Auto scale values
+            let mut scale = *entity_props
+                .backproject_scale
+                .or(&re_data_store::EditableAutoValue::Auto(
+                    tensor.meter.map_or_else(
+                        || match &tensor.data {
+                            TensorData::U16(_) => 1.0 / u16::MAX as f32,
+                            _ => 1.0,
+                        },
+                        |meter| match &tensor.data {
+                            TensorData::U16(_) => 1.0 / meter * u16::MAX as f32,
+                            _ => meter,
+                        },
+                    ),
+                ))
+                .get();
+            let mut radius_scale = *entity_props
+                .backproject_radius_scale
+                .or(&re_data_store::EditableAutoValue::Auto(0.02))
+                .get();
+
+            ui.label("Backproject scale");
+            let speed = (scale * 0.05).at_least(0.01);
+            ui.add(
+                egui::DragValue::new(&mut scale)
+                    .clamp_range(0.0..=1.0e8)
+                    .speed(speed),
+            )
+            .on_hover_text("Scales the backprojected point cloud");
+            ui.end_row();
+
+            ui.label("Backproject radius scale");
+            let speed = (radius_scale * 0.05).at_least(0.01);
+            ui.add(
+                egui::DragValue::new(&mut radius_scale)
+                    .clamp_range(0.0..=1.0e8)
+                    .speed(speed),
+            )
+            .on_hover_text("Scales the radii of the points in the backprojected point cloud");
+            ui.end_row();
+
+            // We actually want these to update anyhow, as this might be the first tick, where we
+            // compute the initial Auto value.
+            entity_props.backproject_scale = EditableAutoValue::UserEdited(scale);
+            entity_props.backproject_radius_scale = EditableAutoValue::UserEdited(radius_scale);
```

Comment:
ah but now we're only updating them when the ui is shown *and* we always mark them as user edited even when there was no edit at all. Both of these things seem problematic.

We should instead update all values that are "auto" in the same place the image plane distance is updated.

The idea is that we later will have the possibility to revert things back to Auto and show them a bit differently depending on whether they are auto or user edited.